### PR TITLE
Away Site Sector Blacklists

### PR DESCRIPTION
**Pull request by RustingWithYou**
Removes several away sites from spawning in Konyang, Tau Ceti, and Burzsia, as they did not make sense to be in the middle of civilised space/in those specific systems.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm [hunk 1]
```diff
@@ -8,7 +8,8 @@
 	spawn_weight = 1
 	suffixes = list("away_site/abandoned_industrial/abandoned_industrial_station.dmm")
 
-	sectors = list(ALL_POSSIBLE_SECTORS)
+	sectors = list(ALL_TAU_CETI_SECTORS, ALL_COALITION_SECTORS, SECTOR_BADLANDS, SECTOR_VALLEY_HALE, SECTOR_GAKAL, SECTOR_LIGHTS_EDGE)
```
Comment (DreamySkrell) [inline maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:11]: why?

File: maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm [hunk 2]
```diff
@@ -8,7 +8,8 @@
...
+	sectors_blacklist = list(SECTOR_TAU_CETI, SECTOR_BURZSIA, SECTOR_HANEUNIM)
```
Comment (DreamySkrell) [inline maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:12]: TC and Haneumin okay, but why Burzsia? It's the "ship graveyard and dumping area".

File: maps/away/away_site/big_derelict/bigderelict.dm [hunk 3]
```diff
@@ -2,7 +2,8 @@
 	name = "large derelict"
 	description = "A very large derelict station. According to the starmap, it shouldn't exist."
 	suffixes = list("away_site/big_derelict/bigderelict.dmm")
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ, ALL_COALITION_SECTORS)
+	sectors = list(SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ, ALL_COALITION_SECTORS)
+	sectors_blacklist = list(SECTOR_BURZSIA, SECTOR_HANEUNIM)
```
Comment (DreamySkrell) [inline maps/away/away_site/big_derelict/bigderelict.dm:6]: Why Burzsia? It's the "ship graveyard and dumping area".

File: maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm (hunk 2 above)
Comment (RustingWithYou) [reply maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:12]: This is a fair point, will change it.

File: maps/away/away_site/big_derelict/bigderelict.dm (hunk 3 above)
Comment (RustingWithYou) [reply maps/away/away_site/big_derelict/bigderelict.dm:6]: this was largely due to the description saying "according to the map, it shouldn't exist". That being said I might change it

File: maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm (hunk 1 above)
Comment (RustingWithYou) [reply maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:11]: using ALL_POSSIBLE_SECTORS is bad, as it means future people will have to go through and blacklist sectors that it doesn't fit individually, as well as existing sectors that won't have a lot of human ruins such as the lemurian sea

Comment (DreamySkrell) [reply maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:11]: It is not bad, and that is fully intentional, and better than the alternative. Reasoning being:
- It is very easy to notice that a site/ship spawns in a sector, when it should not be spawning. You just look at the possible sites in current sector, that I added to show around round start.
- It is very hard to notice that a site/ship should be spawning in a sector, but it does not. You have to go through all the files and folders of ships and sites, and we have a ton of those, and only getting more and more.

The whole point of definitions like `ALL_POSSIBLE_SECTORS` or `ALL_COALITION_SECTORS` is to not have to go through every new ship/site to add them to whatever new sector that is being added. Like how it used to be in the past before I made `sectors` to allow nested lists, and added `sectors_blacklist`.

Comment (DreamySkrell) [reply maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:11]: So yeah. Please keep it `ALL_POSSIBLE_SECTORS` and blacklist from any sectors it should not spawn in.

Also. The blacklist can have nested lists too, so in the future there could be defines like `ALL_SECTORS_WITH_NO_HUMAN_DERELICTS` for example, so any human derelicts would just have a `sectors_blacklist = list(ALL_SECTORS_WITH_NO_HUMAN_DERELICTS)`. Or the same situation with defines like `ALL_CIVILIZED_SECTORS` or something that currently would hold Tau Ceti and Haumenin.

**Review by FluffyGhoster — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**